typed-racket-lib/typed-racket/typecheck/tc-envops.rkt

```diff
@@ -97,12 +122,104 @@
     (pattern (~seq #:unreachable form:expr))
     (pattern (~seq) #:with form #'(begin)))
   (syntax-parse stx
-    [(_ ps:expr u:unreachable? . b)
-     #'(let-values ([(new-env atoms) (env+ (lexical-env) ps)])
+    [(_ ps:expr u:unreachable? body ...)
```

Comment:
I thought 'body ...' might be clearer/harder to mess up (I introduced a bug I had to find when I forgot b was a list). Maybe changing 'b' to 'bodies' and leaving the '.' would be the clarification I _really_ wanted.